db/models/articlesIdPatch.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../connection");
+
+exports.updateArticleById = (article_id, inc_votes) => {
+  const queryStr = `
+    UPDATE articles
+    SET votes = votes + $1
+    WHERE article_id = $2
+    RETURNING *;
+  `;
+  return db
+    .query(queryStr, [inc_votes, article_id])
+    .then(({ rows }) => rows[0]);
```

Comment:
I would recommend opening up this .then(...) arrow function so you can do error handling within here, instead of your controller